### DISCUSSION TITLE
Can you use VNC or similar to connect to a virtual SO Desktop instance

### DISCUSSION THREAD

**Discussion by rosswakelin** [Security-Onion-Solutions/securityonion#13377] [2.4] [answered]
### Version

2.4.80

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

32

### Storage for /

100

### Storage for /nsm

1000

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Once you have installed a SO Desktop successfully on a virtual platform, is there any support for connecting to that desktop remotely e.g. VNC?

Thanks

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment, marked as answer]
You may be able to manually install a VNC server, but we don't provide any support for that.

**Comment by rosswakelin** [comment]
Thanks, I was just concerned if there was anything in the Desktop package that would prevent us doing that.
